plugins/DevicePlugins/DevicePlugins.php

```diff
@@ -56,6 +57,14 @@ public static function getAllPluginColumns()
                 $instance = new $className();
                 $instances[] = $instance;
             }
+
+            /**
+             * Triggered to filter / restrict dimensions.
+             *
+             * @param Dimension[] $dimensions An array of dimensions
+             */
+            Piwik::postEvent('Dimension.filterDimensions', array(&$instances));
```

Comment:
@sgiehl is this event name expected? Or maybe it was meant to remove some dimensions but now it's done in core  and not in plugin? Could call a core method here like `Dimension::removeDisabledDimensions()`

BTW just seeing we support other plugins to add extra device columns? Is this used currently? If not could maybe even just hard code the columns.

Reply:
> BTW just seeing we support other plugins to add extra device columns? Is this used currently? If not could maybe even just hard code the columns.

Using that for at least this plugin: https://github.com/sgiehl/piwik-plugin-DeviceFeatureWebGL
Not sure if anyone else is using it in a plugin....